src/main/scala/uk/gov/nationalarchives/consignmentexport/Validator.scala

```diff
@@ -3,7 +3,8 @@ package uk.gov.nationalarchives.consignmentexport
 import cats.implicits._
 import graphql.codegen.GetConsignmentExport.getConsignmentForExport.GetConsignment
 import graphql.codegen.GetConsignmentExport.getConsignmentForExport.GetConsignment.Files
-import uk.gov.nationalarchives.consignmentexport.Validator.{ValidatedFFIDMetadata, ValidatedFileMetadata}
+import graphql.codegen.GetConsignmentExport.getConsignmentForExport.GetConsignment.Files.AntivirusMetadata
```

Comment:
Think this is an unused import: `import graphql.codegen.GetConsignmentExport.getConsignmentForExport.GetConsignment.Files.AntivirusMetadata`